indico/modules/events/timetable/templates/pdf/_cover_page.html

```diff
@@ -0,0 +1,27 @@
+{% from 'events/display/conference/_util.html' import format_event_date %}
+
+{% macro render_cover_page(event) %}
+    <div class="pagebreak cover">
+        <div class="cover-wrapper">
+            <div>
+                {% if event.logo %}
+                    <img alt="Event logo" src="event://logo" class="logo" />
+                {% endif %}
+                <h1>{{ event.title }}</h1>
+                <h3>{{ format_event_date(event) }}</h3>
+                <h1>{% trans %}Programme{% endtrans %}</h1>
+                {% if event.has_location_info %}
+                    <div>
+                        <h4 class="location">Location:</h4>
```

Comment:
i18n

Reply:
Also, this text looks a bit out of place, especially with the `:` at the end... Maybe it's not needed and we can just get rid of it altogether, since it should be pretty c lear that the text there is the location of the event?

Reply:
Just removed it, I agree with you